src/PublicApiAnalyzers/Microsoft.CodeAnalysis.PublicApiAnalyzers.md

```diff
@@ -14,7 +14,7 @@ All public types and members should be declared in PublicAPI.txt. This draws att
 
 ## [RS0017](https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md): Remove deleted types and members from the declared API
 
-When removing a public type or member the corresponding entry in PublicAPI.txt should also be removed. This draws attention to API changes in the code reviews and source control history, and helps prevent breaking changes.
+When removing a public type or member, put that entry in PublicAPI.Unshipped.txt with '\*REMOVED\*' prefix. This draws attention to API changes in the code reviews and source control history, and helps prevent breaking changes.
```

Comment:
This suggestion removes the "correct" escaping. This should be fixed in the tool generating the doc file.

```suggestion
When removing a public type or member, put that entry in PublicAPI.Unshipped.txt with '*REMOVED*' prefix. This draws attention to API changes in the code reviews and source control history, and helps prevent breaking changes.
```

Reply:
Thank you.